modules/test/playwright/tests/style-book-web/stylebookEditor.spec.ts

```diff
@@ -60,3 +67,160 @@ test('Checks the correct label for restricted pages in the preview selector', as
 		page.getByRole('menuitem', {name: `${pageName} Restricted Page`})
 	).toBeVisible();
 });
+
+test('LPD-35561 Preview StyleBook when edit StyleBook', async ({
+	page,
+	pageEditorPage,
+	pagesAdminPage,
+	productMenuPage,
+	site,
+	styleBooksPage,
+}) => {
+	await test.step('Add a content page', async () => {
+		await productMenuPage.openProductMenuIfClosed();
+
+		await productMenuPage.goToPages();
+
+		await pagesAdminPage.createNewPage({
+			addButtonLabel: 'Page',
+			draft: true,
+			name: 'Test Page Name',
+			template: 'Blank',
+		});
+	});
+
+	await test.step('Add a Banner Center to page', async () => {
+		await pageEditorPage.goToSidebarTab('Fragments and Widgets');
+
+		await pageEditorPage.addFragment(
+			'Featured Content',
+			'Banner Center',
+			page.locator('div.page-editor__root')
```

Comment:
It's better to not add the 3rd param of the dropzone. It's optional and less likely to break if you don't add it.

Reply:
I remember why I had to do this instead of use `addFragment`. There is an error in Page Editor with the Banner Center fragment when using `Enter + Enter` to deploy it.

![page-editor-enter-enter](https://github.com/user-attachments/assets/974fba4c-7eba-474b-a850-e938edcde7e8)

Reply:
I created an issue for this at https://liferay.atlassian.net/browse/LPD-39539